lib/interface_quda.cpp

```diff
@@ -151,8 +150,8 @@ std::vector< std::vector<ColorSpinorField*> > chronoResident(QUDA_MAX_CHRONO);
 static int *num_failures_h = nullptr;
 static int *num_failures_d = nullptr;
 
-cudaDeviceProp deviceProp;
-cudaStream_t *streams;
+qudaDeviceProp deviceProp;
```

Comment:
Likely it would be more appropriate to move the device properties variable to the cuda target file as opposed to having it here.  Similarly the logic regarding enumeration of devices and querying properties should all be moved into a CUDA specific file, since this likely doesn't abstract simply like this.  This allows for more target specific device setup.